recipes/mingw/all/conanfile.py

```diff
@@ -0,0 +1,73 @@
+import os
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class MingwConan(ConanFile):
+    name = "mingw"
```

Comment:
This is a recipe for a fork of mingw, mingw-w64 (see wikipedia: https://en.wikipedia.org/wiki/MinGW)
So I propose to rename it to `mingw-w64`.
```suggestion
    name = "mingw-w64"
```

Reply:
especially with MinGW coming back to life on https://osdn.net/projects/mingw/

Reply:
Great find! I'll update it, thanks

Reply:
Done